package.json

```diff
@@ -11,7 +11,8 @@
     "test": "jest",
     "lint": "eslint --ignore-path .gitignore --ext .js,.jsx,.ts,.tsx --color .",
     "prettier": "prettier --ignore-path .gitignore '**/*.+(js|jsx|json|ts|tsx|md|yml|yaml)'",
-    "postinstall": "patch-package",
+    "prepack": "rimraf *.tgz",
+    "postinstall": "patch-package && node post-install-setup.js",
```

Comment:
This will run after every install right? Maybe you can add some form of check to see if this setup has already be done, that way on multiple runs it will actually only run once

Reply:
I think that can be optimized for sure. I am assuming `npm install` or `npm ci` is not something that developer runs very frequently, so I didn't think the overhead was too much. I've added to the existing postinstall, since I am not sure what `patch-package` is used for. I will add some extra checks to allow for caching